src/nni_manager/training_service/remote_machine/gpuScheduler.ts

```diff
@@ -93,6 +95,16 @@ export class GPUScheduler {
             rmMeta.gpuReservation.forEach((reserveTrialJobId : string, gpuIndex : number) => {
                 if (reserveTrialJobId === trialJobId) {
                     rmMeta.gpuReservation.delete(gpuIndex);
+                    let occupiedTrialNumPerGPU = this.machineOccupiedTrialNumPerGPUMap.get(rmMeta);
+                    if(occupiedTrialNumPerGPU === undefined) {
+                        throw new Error(`gpu scheduler error!`);
```

Comment:
I am not sure the exception contains enough information to debug...

Reply:
fixed.